aiosmtpd/main.py

```diff
@@ -134,10 +140,13 @@ def main(args=None):
     if args.debug > 2:
         loop.set_debug(enabled=True)
 
-    log.info('Server listening on %s:%s', args.host, args.port)
-
-    server = loop.run_until_complete(
-        loop.create_server(factory, host=args.host, port=args.port))
+    if args.supervised:
+        log.info('Server listening on socket-activated file descriptor')
+        stage = loop.create_server(factory, sock=socket.fromfd(3, socket.AF_INET, socket.SOCK_STREAM))
```

Comment:
It looks like these are the lines that aren't actually getting tested, according to what I can glean from Travis.

Reply:
https://travis-ci.com/aio-libs/aiosmtpd/jobs/248519180#L566